samples/getting-started/android/app/build.gradle

```diff
@@ -3,11 +3,11 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
```

Comment:
@rikner do we need to do anything regarding tagged pointers here too or is that only from v31+?

Reply:
ah sure right! we must disable tagged pointers in the sample apps manifest, see https://github.com/flowkey/NativePlayer/pull/898/files#diff-dadb150a88f63da91485cd57b79259d15a19bb20782f0f0bb3795237b926171bR14
otherwise we would risk crashes.

Reply:
ok, would you be able to make a PR for that? (not urgent)